package.json

```diff
@@ -11,43 +11,43 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^12.1.1",
-    "@angular/cdk": "^12.1.1",
-    "@angular/common": "~12.1.1",
-    "@angular/compiler": "~12.1.1",
-    "@angular/core": "~12.1.1",
-    "@angular/forms": "~12.1.1",
-    "@angular/material": "^12.1.1",
-    "@angular/platform-browser": "~12.1.1",
-    "@angular/platform-browser-dynamic": "~12.1.1",
-    "@angular/router": "~12.1.1",
+    "@angular/animations": "^14.0.0" ,
+    "@angular/cdk": "^14.0.0",
+    "@angular/common": "^14.0.0",
+    "@angular/compiler": "^14.0.0",
+    "@angular/core": "^14.0.0",
+    "@angular/forms": "^14.0.0",
+    "@angular/material": "^14.0.0",
+    "@angular/platform-browser": "^14.0.0",
+    "@angular/platform-browser-dynamic": "^14.0.0",
+    "@angular/router": "^14.0.0",
     "bootstrap": "^4.3.1",
     "core-js": "^3.1.3",
-    "ng-dynamic-breadcrumb": "^6.0.0",
-    "ng7-bootstrap-breadcrumb": "^6.0.0",
-    "ng7-dynamic-breadcrumb": "^6.0.0",
-    "ng7-mat-breadcrumb": "^6.0.0",
+    "ng-dynamic-breadcrumb": "projects/ng-dynamic-breadcrumb",
```

Comment:
I changed these to project based references as I can not upgrade the external deps myself.